src/mission_ui.cpp

```diff
@@ -102,7 +103,16 @@ void game::list_missions()
         if( selection < umissions.size() ) {
             const auto miss = umissions[selection];
             const nc_color col = u.get_active_mission() == miss ? c_light_green : c_white;
-            int lines = fold_and_print( w_missions, 3, 31, getmaxx( w_missions ) - 33, col, miss->name() );
+            std::string for_npc = to_string( miss->get_npc_id() );
+            if( miss->get_npc_id() >= 0 ) {
+                npc *guy = g->find_npc( miss->get_npc_id() );
+                if( guy ) {
+                    for_npc = string_format( _( "for %s " ), guy->disp_name() );
```

Comment:
I wonder if we need the space after `%s`?

Reply:
No, typo.